CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -111,6 +118,9 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 				newOffset += velocity.y > 0 ? pageHeight : -pageHeight
 			}
 			return CGPoint(x: proposedContentOffset.x, y: newOffset)
+            
+        default:
+            return .zero
```

Comment:
The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?